docs/en/latest/deployments/kind.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Install Ingress APISIX on Kind
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+This document explains how to install Ingress APISIX on [kind](https://kind.sigs.k8s.io/).
+
+## Prerequisites
+
+* Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+* Install [Helm](https://helm.sh/).
```

Comment:
Need to specify the versions of `kind` and `helm`.

Reply:
I don't think we need to specify that GO/Docker etc. are needed, they are actually kind dependencies.

We just need to note that you need to properly install kind and use it to create a Kubernetes cluster.

Reply:
After considering, I agree.